pkg/aspect/accessLogsManager_test.go

```diff
@@ -54,36 +74,20 @@ func TestAccessLoggerManager_NewAspect(t *testing.T) {
 		aspect: tl,
 	}
 
-	// TODO: add back tests for custom when we introduce descriptors
-	//customExec := &accessLogsExecutor{
-	//	name:   "custom_access_log",
-	//	aspect: tl,
-	//	labels: map[string]string{"test": "test"},
-	//}
-
 	combinedStruct := &aconfig.AccessLogsParams{
 		LogName: "combined_access_log",
 		Log: &aconfig.AccessLogsParams_AccessLog{
-			LogFormat: aconfig.COMBINED,
+			DescriptorName: "common",
```

Comment:
Since common log and combined log are known log formats, can we add a test that ensures that our packaged default descriptors actually  produce the correct log entries?

We should also agree on a mechanism to provide default descriptors. I think mixer config manager with packaged default set of descriptors as yml files. (like we have in testdata/ is a reasonable start)